openstack/barbican/templates/service.yaml

```diff
@@ -16,6 +16,7 @@ metadata:
     prometheus.io/scrape: "true"
     prometheus.io/port: {{.Values.statsd.port | quote }}
     prometheus.io/targets: {{ .Values.alerts.prometheus | quote }}
+    {{- include "utils.linkerd.pod_and_service_annotation" . | indent 8 }}
```

Comment:
Does that indentation level look correct in the rendered chart? I'd assume it should be something like `4` instead.

Reply:
helm-diff worked with 8, however reducing it to 4 instead.